src/main/sensors/battery.c

```diff
@@ -79,6 +79,7 @@ PG_RESET_TEMPLATE(batteryConfig_t, batteryConfig,
 
     .voltage = {
         .scale = VBAT_SCALE_DEFAULT,
+        .cellDetect = 430,
```

Comment:
Please remember to increase version when adding new parameters to PGs
` PG_REGISTER_WITH_RESET_TEMPLATE(batteryConfig_t, batteryConfig, PG_BATTERY_CONFIG, 1);`

Reply:
Indeed I forgot about the PG version. I think we could also set `vbat_max_cell_voltage` default to 4.20V now since it is not used for the cell count detection anymore and lower `VBATT_CELL_FULL_MAX_DIFF` default accordingly to 0.10V.